Index: www/js/app.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- www/js/app.js	(revision 024bad0795cfb314ed00e87ca2228b88b62e25ae)
+++ www/js/app.js	(revision )
@@ -15,7 +15,7 @@
       cordova.plugins.Keyboard.hideKeyboardAccessoryBar(true);
     }
     if (window.StatusBar) {
-      // org.apache.cordova.statusbar required
+      // org.apacrhe.cordova.statusbar required
       StatusBar.styleDefault();
     }
   });
